src/db/protocol.h

```diff
@@ -11,20 +11,15 @@ Db Protocol Handler
 Functions
 ***********************************************************************************************************************************/
 // Process db protocol requests
-FN_EXTERN void dbOpenProtocol(PackRead *param, ProtocolServer *server);
-FN_EXTERN void dbQueryProtocol(PackRead *param, ProtocolServer *server);
-FN_EXTERN void dbCloseProtocol(PackRead *param, ProtocolServer *server);
+FN_EXTERN ProtocolServerResult *dbOpenProtocol(PackRead *param);
+FN_EXTERN ProtocolServerResult *dbQueryProtocol(PackRead *param, void *sessionData);
 
 /***********************************************************************************************************************************
 Protocol commands for ProtocolServerHandler arrays passed to protocolServerProcess()
 ***********************************************************************************************************************************/
-#define PROTOCOL_COMMAND_DB_OPEN                                    STRID5("db-o", 0x7ec440)
-#define PROTOCOL_COMMAND_DB_QUERY                                   STRID5("db-q", 0x8ec440)
-#define PROTOCOL_COMMAND_DB_CLOSE                                   STRID5("db-c", 0x1ec440)
+#define PROTOCOL_COMMAND_DB                                         STRID5("db", 0x440)
 
 #define PROTOCOL_SERVER_HANDLER_DB_LIST                                                                                            \
-    {.command = PROTOCOL_COMMAND_DB_OPEN, .handler = dbOpenProtocol},                                                              \
-    {.command = PROTOCOL_COMMAND_DB_QUERY, .handler = dbQueryProtocol},                                                            \
-    {.command = PROTOCOL_COMMAND_DB_CLOSE, .handler = dbCloseProtocol},
```

Comment:
Because close is implicit. When the session is closed the server will automatically free whatever session object it was given. That free closes the connection to Postgres. Open still needs to be explicit.